src/Formio.js

```diff
@@ -550,7 +550,7 @@ class Formio {
                 if (uploadStartCallback) {
                   uploadStartCallback();
                 }
-                return provider.uploadFile(file, fileName, dir, progressCallback, url, options, fileKey, groupPermissions, groupId);
+                return provider.uploadFile({ storage, file, fileName, dir, progressCallback, url, options, fileKey, groupPermissions, groupId, abortCallback });
```

Comment:
Isn't this still changing the function signature? To accept an object instead of arguments?...  If so, then we cannot change function signatures within a major version. ( we can do this in 5.x but not 4.x).

Reply:
I see
Thought we must keep it untouched only in Formio
Will change it back in provider files too then